partcad/src/partcad/ai_openai.py

```
@@ -37,14 +36,13 @@
 }
 
 
-def openai_once():
+def openai_once(latest_key: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `openai_once`

Add return type annotation: `bool`

(ANN201)